styles.css

```diff
@@ -98,9 +114,17 @@ body {
 }
 
 .layout-container {
+  display: grid;
+  grid-template-columns: repeat(3, 200px);
```

Comment:
Same as in exercise 1, When building a responsive app try to avoid using fixed sizes like 200px, try using the following code instead:

  grid-template-columns: repeat(3, 1fr);
  grid-template-rows: repeat(2, 1fr);